swap_meet/item.py

```diff
@@ -1,2 +1,15 @@
 class Item:
-    pass
\ No newline at end of file
+    CONDITION_DESCRIPTIONS = {0:"Poor", 1: "Okay", 2: "Fair", 3: "Good", 4: "Excellent", 5: "Pristine"}
```

Comment:
This line is pretty long, I suggest formatting the dictionary a little differently to make it easier to read, even though it take more lines:
```suggestion
    CONDITION_DESCRIPTIONS = {
        0:"Poor", 
        1: "Okay", 
        2: "Fair", 
        3: "Good", 
        4: "Excellent", 
        5: "Pristine"
    }
```